Makefile

```diff
@@ -3,10 +3,13 @@ VERSION ?= latest
 # From https://github.com/VictoriaMetrics/operator/pull/44
 YAML_DROP=$(YQ) delete --inplace
 YAML_DROP_PREFIX=spec.validation.openAPIV3Schema.properties.spec.properties
-
+ 
 # Produce CRDs that work back to Kubernetes 1.11 (no version conversion)
 CRD_OPTIONS ?= "crd:trivialVersions=true"
 
+# This is ultimately the previous tag however at the time of running the Makefile it would be the latest
+GITHUB_LATEST_TAG := $(strip $(shell git tag --sort=committerdate | tail -1))
```

Comment:
At the time of running `release.yml` workflow, I think we already have the latest version tag. Probably we'd better use `tail -n 2 | head -n 1`?

Reply:
ah, maybe this is my lack of Makefile knowledge, are the global variables not ran before any targets are ran? So we run `make github-release` and the global variables are populated first? As a result they are ran before the `ghr ${VERSION} release/` step happens meaning the new release hasn't be created yet?

Reply:
AFAIK our usage of GitHub actions workflow ensures that the job is triggered only after we manually tagged the new release.

So, the global variable is populated first, earlier than the tasks. But it still `tail -1` would return the tag just created manually, where you want seems like the previous one of that.